src/brevitas/graph/equalize.py

```diff
@@ -361,7 +492,18 @@ def _is_supported_module(graph_model: GraphModule, node: Node) -> bool:
 
 def _is_scale_invariant_module(graph_model: GraphModule, node: Node) -> bool:
     return node.op == 'call_module' and isinstance(
-        get_module(graph_model, node.target), _scale_invariant_layers)
+        get_module(graph_model, node.target),
+        _scale_invariant_layers + _scale_invariant_activations)
+
+
+def _is_scale_invariant_activation(graph_model, node):
+    return node.op == 'call_module' and isinstance(
```

Comment:
shouldn't we check functionals too?

Reply:
The original act eq was based on FX and potentially the FunctionalToModule transformation.
I guess it's too strong of an assumption